plugins/com.google.cloud.tools.eclipse.appengine.validation/src/com/google/cloud/tools/eclipse/appengine/validation/XmlValidationHelper.java

```diff
@@ -23,9 +23,9 @@
 interface XmlValidationHelper {
 
   /**
-   * Searches the {@link Document} for banned elements and returns them
-   * as a list of {@link BannedElement}s.
+   * Searches the {@link Document} for element problems and returns them
+   * as a list of {@link ElementProblem}s.
    */
-  ArrayList<BannedElement> checkForElements(IResource resource, Document document);
+  ArrayList<ElementProblem> checkForElements(IResource resource, Document document);
```

Comment:
Maybe this should be renamed to `checkForProblems`?

Reply:
done